docs/application/native/api/iot-headed/5.5/group__Evas__Object__Box__Group.html

```diff
@@ -1821,7 +1854,7 @@
 <p>Set the default box <em>api</em> struct (Evas_Object_Box_Api) with the default values. May be used to extend that API.</p>
 <dl class="params"><dt><b>Parameters:</b></dt><dd>
   <table class="params">
-    <tr><td class="paramname">api</td><td>The box API struct to set back, most probably with overridden fields (on class extensions scenarios) </td></tr>
+    <tr><td class="paramname">api</td><td>The box API struct to set back, most probably with overridden fields (on class extensions scenarios)  2.3 </td></tr>
```

Comment:
Please check the updates like this at here and line 1836.

Reply:
checked

Reply:
https://review.tizen.org/gerrit/#/c/platform/upstream/efl/+/250985/